src/java/picard/util/IntervalListTools.java

```diff
@@ -32,9 +32,27 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = " General tool for manipulating interval lists, " +
-                "including sorting, merging, padding, uniqueifying, and other set-theoretic operations. Default operation if given one or more inputs is to " +
-                "merge and sort them.  Other options are controlled by arguments.",
+        usage = "General tool for manipulating interval lists, including sorting, merging, padding, customizing, " +
+                "and other set-theoretic operations. Default operation if given one or more inputs is to merge" +
+                " and sort them.  Other options are controlled by arguments.<br /><br />" +
+
+                "Lists intervals with respect to a reference sequence that can be written to and read from a file." +
```

Comment:
I'm not sure what this is saying or how it fits with the rest. Is this supposed to provide a definition of what interval lists are? Is it saying that the tool can list intervals?